Makefile

```diff
@@ -1,3 +1,8 @@
+# Git
+GIT=git
+GIT_SUBMODULES=$(shell sed -nE 's/path = +(.+)/\1\/.git/ p' .gitmodules | paste -s -)
```

Comment:
Don't do that. You should make a list of all available git sub-modules. In this case, there is only one.